ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -448,11 +445,12 @@ impl Blockstore {
     fn purge_special_columns_exact(
         &self,
         batch: &mut WriteBatch,
-        from_slot: Slot,
-        to_slot: Slot, // Exclusive
+        from_slot: Slot, // inclusive
+        to_slot: Slot,   // inclusive
```

Comment:
For consistency with other functions, maybe mention `[from_slot, to_slot]` in the function doc string, which would then make these redundant.